server/repository/src/db_diesel/insurance_provider_row.rs

```diff
@@ -63,6 +63,15 @@ impl<'a> InsuranceProviderRowRepository<'a> {
         Ok(result)
     }
 
+    pub fn find_all_active(&self) -> Result<Vec<InsuranceProviderRow>, RepositoryError> {
+        let result = insurance_provider::table
+            .load(self.connection.lock().connection())?
+            .into_iter()
+            .filter(|row: &InsuranceProviderRow| row.is_active == true)
+            .collect();
+        Ok(result)
+    }
+
```

Comment:
Also option to remove the now unused find_all if that would not be confusing.

Reply:
I think it's ok to leave it.